### PR TITLE
Updating prices.usd with Covalent new token CXT

### PR DESCRIPTION
**Pull request by Krish-vemula**
### Update!
Please build spells in the proper [subproject](../dbt_subprojects/) directory. For more information, please see the main [readme](../README.md), which also links to a GH discussion with the option to ask questions.

### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [X] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name*
- **Description:** [Detailed description of the new spell(s) and their purpose]
- **Who are the new spell(s) for?** [Internal team or general community]
- **How will the new spell(s) be used downstream?** [Description of downstream usage]
- **Implementation details:** [Information on how the spell(s) are implemented]
- **Test instructions:** [How to test the new spell(s)]
- **Related issue(s):** [Link to related issues, if any]

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]
This PR is to add Covalent X Token (CXT) to the prices.usd , we have our old token CQT(Covalent Query Token) on this but we migrated our token from CQT to CXT 
---

### For bug fixes
If you are fixing a bug, please provide the following information:

- **Description:** [Description of the bug fix]
- **Steps to reproduce:** [How to reproduce the bug]
- **Implementation details:** [Information on how the bug was fixed]
- **Test instructions:** [How to test the fix]
- **Related issue(s):** [Link to related issues, if any]

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

**Comment by Krish-vemula** [comment]
I have read the CLA Document and I hereby sign the CLA
recheck

**Comment by Krish-vemula** [comment]
recheck

**Comment by Krish-vemula** [comment]
I have read the CLA Document and I hereby sign the CLA

File: dbt_subprojects/tokens/models/prices/ethereum/prices_ethereum_tokens.sql [hunk 1]
```diff
@@ -874,6 +874,7 @@ FROM
     ('kuma-kuma-inu', 'ethereum', 'KUMA', 0x48c276e8d03813224bb1e55f953adb6d02fd3e02, 18),
     ('cor-coreto', 'ethereum', 'COR', 0x9c2dc0c3cc2badde84b0025cf4df1c5af288d835, 18),
     ('cqt-covalent', 'ethereum', 'CQT', 0xd417144312dbf50465b1c641d016962017ef6240, 18),
+    ('cxt-covalent-x-token', 'ethereum', 'CXT', 0x7ABc8A5768E6bE61A6c693a6e4EAcb5B60602C4D, 18),
```
Comment (Hosuke) [inline dbt_subprojects/tokens/models/prices/ethereum/prices_ethereum_tokens.sql:877]: https://etherscan.io/token/0x7abc8a5768e6be61a6c693a6e4eacb5b60602c4d

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @Krish-vemula

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**